ql/src/java/org/apache/hadoop/hive/ql/exec/SerializationUtilities.java

```diff
@@ -278,7 +284,7 @@ public Kryo create() {
    * @return kryo instance
    */
   public static Kryo borrowKryo() {
-    Kryo kryo = kryoPool.borrow();
+    Kryo kryo = kryoPool.obtain();
```

Comment:
Maybe rename the method itself here as well?  something like obtainKryo() ?

Reply:
it's very unfortunate that kryo changed this method name without any reasons (or just I don't understand that :) )...we might want a method name that reflects the behavior of "getting a kryo instance from the pool", I don't have a strong opinion about that, but I'm not sure if we need to change a public method name because kryo changed theirs